libs/server-sent-events/src/client.cpp

```diff
@@ -341,13 +341,21 @@ class FoxyClient : public Client,
         // If any backoff is taking place, cancel that as well.
         backoff_timer_.cancel();
```

Comment:
This function can theoretically throw as well, but I cannot find any confirmation of a condition where it actually throws.